src/shared-components/PeopleTabNavbarButton/PeopleTabNavbarButton.tsx

```
@@ -0,0 +1,69 @@
+  title,
+  icon,
+  isActive,
+  action,
+  testId,
+}) => {
+  return (
+    <div
+      onClick={action}
+      className={`${styles.peopleTabBtnBlock} ${isActive ? styles.peopleTabActiveButton : ''}`}
+      data-testid={testId}
+    >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)